### PR TITLE
Gives honey antibiotic properties

### PR DESCRIPTION
**Pull request by 81Denton**
:cl: Denton
tweak: Did you know that honey has antibiotic properties?
/:cl:

I'm trying to find a way to give bees more purpose than "inject skewium and open the door xd". This is fairly minor since few people do surgery, but IMO a step in the right direction.

Why the high success multiplier? This is 14% more than bacchus' blessing, which is much easier and faster to obtain compared to having to deal with the QM and security.

### CODE REVIEW

**Comment by optimumtact** [comment]
\>try to base your PR on reality
\>your reagent is unrealistically overpowered

how's that working out for you fam?

**Comment by 81Denton** [comment]
Using realism was a mistake, as always

I wouldn't call the reagent overpowered if you consider that sterilizine (20%) is available in every single nanomed/wallmed and bacchus' blessing (~46%) can be mixed without any hassle - just grind a can of spraypaint or ask chemistry for ethanol and borrow some enzyme from the chef.
That's compared to ordering bees from cargo, which won't happen most times since cargo is either busy buying nullcrates or has PTSD from griff bees.

**Comment by fludd12** [comment]
"Oh, you want surgery? Let me dip you in honey first, then we can get started. It aids the process."

"..."

"What? Of course I'm a doctor. Don't you see my lab coat and doctoral BEES?"

"..."

"Of course that's a thing. Now lie down and let me cover you in honey for my amusement."

**Comment by ExcessiveUseOfCobblestone** [comment]
@optimumtact I think this is Karma for all the times you posted Bee Movie Script on the forums.

File: code/modules/reagents/chemistry/reagents/food_reagents.dm [hunk 1]
```diff
@@ -571,6 +571,14 @@
 		M.adjustToxLoss(-1*REM, 0)
 	..()
 
+/datum/reagent/consumable/honey/reaction_mob(mob/living/M, method=TOUCH, reac_volume)
+  if(iscarbon(M) && (method in list(TOUCH, VAPOR, PATCH)))
+    var/mob/living/carbon/C = M
+    for(var/s in C.surgeries)
+      var/datum/surgery/S = s 
+      S.success_multiplier = max(0.60, S.success_multiplier) // +60% success probability on each step, compared to bacchus' blessing's ~46%
```
Comment (ShizCalev) [inline code/modules/reagents/chemistry/reagents/food_reagents.dm]: why not just 0.6?

Comment (81Denton) [reply code/modules/reagents/chemistry/reagents/food_reagents.dm]: True

**Review by ShizCalev — APPROVED** [review]

**Comment by optimumtact** [comment]
on the balance of it I'm inclined to bee-lieve the comment of denton that this is adequately balanced

**Pull request merged**